Problem with impoting ldap_ou

Hi, I tried to use this provider and I encounter on problem with importing objects to state.

Terraform version: 1.4.2
Ldap provider version: 0.8.6 (I also tried 0.8.5 with the same error)

Here is code I used:

1. My provider configuration (I tried tls_insecure - true/false and it's not working)

```tf
provider "ldap" {
  host               = "x.x.x.x.x.x"
  port               = 636
  bind_user          = "cn=admin,dc=x,dc=x,dc=x,dc=x,dc=x,dc=x"
  bind_password      = "admin"
  tls                = true
  tls_insecure       = false
}
```

2. Resource configuration:

```tf
resource "ldap_ou" "example" {
  name = "example"
  ou   = "ou=Clients,ou=Roles,dc=x,dc=x,dc=x,dc=x,dc=x,dc=x"
}
```

3. import command

```bash
terraform import -var-file="input-variables.json" "ldap_ou.example" "ou=example,ou=Clients,ou=Roles,dc=x,dc=x,dc=x,dc=x,dc=x,dc=x"
```

4. And output

```
ldap_ou.example: Importing from ID "ou=example,ou=Clients,ou=Roles,dc=x,dc=x,dc=x,dc=x,dc=x,dc=x"...
ldap_ou.example: Import prepared!
  Prepared ldap_ou for import
...
╷
│ Error: Request cancelled
│ 
│ The plugin.(*GRPCProvider).ReadResource request was cancelled.
╵


Stack trace from the terraform-provider-ldap_v0.8.6 plugin:

panic: runtime error: index out of range [0] with length 0

goroutine 46 [running]:
github.com/Ouest-France/terraform-provider-ldap/ldap.resourceLDAPOURead({0xd15888, 0xc00060f320}, 0xc000124f00, {0xbc6600?, 0xc0000933e0})
        github.com/Ouest-France/terraform-provider-ldap/ldap/resource_ldap_ou.go:106 +0x766
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xc0003d62a0, {0xd158c0, 0xc000489e30}, 0xd?, {0xbc6600, 0xc0000933e0})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.1/helper/schema/resource.go:724 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc0003d62a0, {0xd158c0, 0xc000489e30}, 0xc000330820, {0xbc6600, 0xc0000933e0})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.1/helper/schema/resource.go:1015 +0x585
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc00046eb70, {0xd158c0?, 0xc000489d10?}, 0xc00030c480)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.24.1/helper/schema/grpc_provider.go:613 +0x4a5
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadResource(0xc00017c320, {0xd158c0?, 0xc000489530?}, 0xc00060f020)
        github.com/hashicorp/terraform-plugin-go@v0.14.1/tfprotov5/tf5server/server.go:748 +0x4b1
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler({0xbc6a20?, 0xc00017c320}, {0xd158c0, 0xc000489530}, 0xc00034c700, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.14.1/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:349 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000486000, {0xd192a0, 0xc00031a9c0}, 0xc0000c0b40, 0xc000488630, 0x11856f0, 0x0)
        google.golang.org/grpc@v1.50.1/server.go:1340 +0xd23
google.golang.org/grpc.(*Server).handleStream(0xc000486000, {0xd192a0, 0xc00031a9c0}, 0xc0000c0b40, 0x0)
        google.golang.org/grpc@v1.50.1/server.go:1713 +0xa2f
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        google.golang.org/grpc@v1.50.1/server.go:965 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.50.1/server.go:963 +0x28a

Error: The terraform-provider-ldap_v0.8.6 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

5. Logs from ldap server:

```
642544ce conn=1990 fd=24 ACCEPT from IP=x.x.x.x:42792 (IP=0.0.0.0:636)
642544ce conn=1990 fd=24 TLS established tls_ssf=128 ssf=128
642544ce conn=1990 op=0 BIND dn="cn=admin,dc=x,dc=x,dc=x,dc=x,dc=x,dc=x" method=128
642544ce conn=1990 op=0 BIND dn="cn=admin,dc=x,dc=x,dc=x,dc=x,dc=x,dc=x" mech=SIMPLE ssf=0
642544ce conn=1990 op=0 RESULT tag=97 err=0 text=
642544ce conn=1990 op=1 SRCH base="ou=example,ou=Clients,ou=Roles,dc=x,dc=x,dc=x,dc=x,dc=x,dc=x" scope=0 deref=0 filter="(objectClass=organizationalUnit)"
642544ce conn=1990 op=1 SRCH attr=*
642544ce conn=1990 op=1 SEARCH RESULT tag=101 err=0 nentries=1 text=
642544ce conn=1990 fd=24 closed (connection lost)
```

Does anyone have an idea what am i doing wrong?

--- a/ldap/resource_ldap_ou.go
+++ b/ldap/resource_ldap_ou.go
@@ -103,7 +103,7 @@ func resourceLDAPOURead(ctx context.Context, d *schema.ResourceData, m interface
 	}
 
 	// Remove the `OU=<ou-name>,` from the DN to get the OU
-	ou := strings.ReplaceAll(dn, fmt.Sprintf("OU=%s,", attributes["name"][0]), "")
+	ou := strings.Replace(dn, fmt.Sprintf("OU=%s,", nameAttr[0]), "", 1)
 	if err := d.Set("ou", ou); err != nil {
 		return diag.FromErr(err)
 	}

